sdccc/src/test/java/com/draeger/medical/sdccc/tests/biceps/direct/DirectMessageModelAnnexTestTest.java

```diff
@@ -70,6 +73,8 @@ void setUp() throws IOException {
             protected void configure() {
                 bind(TestClient.class).toInstance(testClient);
                 bind(MessageGeneratingUtil.class).toInstance(messageGeneratingUtil);
+                bind(Key.get(Boolean.class, Names.named(TestSuiteConfig.SUMMARIZE_MESSAGE_ENCODING_ERRORS)))
```

Comment:
Instead of defining the same binding over and over again, you should define the default binding in DefaultTestSuiteConfig.

Reply:
Great Idea! I didn't know there was a central place like that...
Done.

Reply:
Ups, I was not expecting that there are even multiple central places!
Moved it to DefaultTestSuiteConfig.